apps/emqx_modules/src/emqx_delayed.erl

```diff
@@ -252,6 +261,16 @@ post_config_update(_KeyPath, _ConfigReq, NewConf, _OldConf, _AppEnvs) ->
     Enable = maps:get(enable, NewConf, undefined),
     load_or_unload(Enable).
 
+clean_by_clientid(ClientId) ->
+    Nodes = mria_mnesia:running_nodes(),
+    _ = [emqx_delayed_proto_v2:clean_by_clientid(Node, ClientId) || Node <- Nodes],
```

Comment:
Maybe:

```suggestion
    lists:foreach(fun(Node) -> emqx_delayed_proto_v2:clean_by_clientid(Node, ClientId) end, Nodes),
```

Reply:
Alternatively, since a new `proto` module was just created here, maybe that could use `erpc:multicall` instead of serial `rpc:call` invocations?